src/deprecation.h

```diff
@@ -8,7 +8,7 @@
 // Deprecation policy:
 // * Shut down 18 weeks' worth of blocks after the estimated release block height.
```

Comment:
[DOES NOT BLOCK] Looks like we missed #2686 for this release.

Reply:
Oh, I was going to do that in the actual 1.0.13 release PR, but I can trivially do it in this PR instead (just requires one more commit).